src/video/sdl2_v.cpp

```diff
@@ -282,7 +282,9 @@ static const SDLVkMapping _vk_mapping[] = {
 	AM_UP(SDLK_F1, SDLK_F12, WKC_F1, WKC_F12),
 
 	/* Numeric part. */
-	AM(SDLK_KP_0, SDLK_KP_9, '0', '9'),
+	/* In SDL2 the order of the numpad keycodes changed so that the smallest value is SDLK_KP_1 and the largest SDLK_KP_0, we need to bind 1-9 and then 0 separately. */
+	AM(SDLK_KP_1, SDLK_KP_9, '1', '9'),
+	AS(SDLK_KP_0,        '0'),
```

Comment:
```suggestion
	AM(SDLK_KP_1, SDLK_KP_9, '1', '9'),
	/* SDL goes from SDLK_KP_1 to SDLK_KP_0, where we go from 0 .. 9. */
	AS(SDLK_KP_0,        '0'),
```

Or something. But your current comment describes a change, rather than the situation. And that means it only makes sense in the context of this PR, but not when you look at the file in 10 years :) I know, a bit of a nuance, but I know how frustrating this kind of code can be in 10 years :P

Btw, nice catch, tnx for fixing :)

Reply:
Given the order might change... should we not just list each one individually?

Reply:
I could do that, and then it would also remove the need for that comment.